console/console-init/ui/src/modules/iot-device/components/ViewMetaData/MetaDataReviewRows.tsx

```diff
@@ -18,7 +18,7 @@ const MetaDataReviewRows: React.FC<IMetaDataReviewRowProps> = ({
     <>
       {values.map((val, index) => (
         <>
-          <MetaDataReviewRow value={val} prevKey={prevkey} index={index} />
+          <MetaDataReviewRow value={val} prevKey={prevkey} />
           {val.type === "array" || (val.type === "object" && <br />)}
```

Comment:
please use enum type `DataType` which is available in constants.ts  for compare value `object` and `array`